components/FlexibleHeader/src/MDCFlexibleHeaderView.h

```diff
@@ -335,6 +335,15 @@ IB_DESIGNABLE
 @property(nonatomic) MDCFlexibleHeaderShiftBehavior shiftBehavior;
 
 /**
+ Hides the view by changing its alpha when the header shifts. Note that this only happens when the
+ header shifting behavior is set to MDCFlexibleHeaderShiftBehaviorEnabled.
+ */
+- (void)hideViewWhenShifted:(nonnull UIView *)view;
```

Comment:
Can we make this a BOOL property?

Reply:
huh?

Reply:
https://www.youtube.com/watch?v=kfVsfOSbJY0